### PR TITLE
Master wallet card update

### PR DESCRIPTION
**Pull request by enjeyw**
**Describe the Pull Request**

Swaps the master wallet card display over to showing the total amount that's in circulation and has been withdrawn, rather than just the amount disbursed. It's a lot more operationally useful because the amount disbursed doesn't take into account reclamations etc.

image

**Todo**

- [ ] Link relevant [trello card](https://trello.com/b/PA2LhlOh/sempo-dev) or [related issue](https://github.com/teamsempo/SempoBlockchain/issues) to the pull request
- [ ] Request review from relevant @person or team
- [ ] QA your pull request. This includes running the app, login, testing changes etc.
- [ ] Bump [backend](https://github.com/teamsempo/SempoBlockchain/blob/e3eb0f480e86d5a8ef2c1814127b70ff018671c4/config.py#L7) and/or [frontend](https://github.com/teamsempo/SempoBlockchain/blob/e3eb0f480e86d5a8ef2c1814127b70ff018671c4/app/package.json#L3) versions following Semver

### CODE REVIEW

File: app/server/utils/metrics/filters.py [hunk 1]
```diff
@@ -143,6 +143,18 @@ def _apply_ca_filters(query, filters, user_join_condition):
     CreditTransfer.transfer_subtype == TransferSubTypeEnum.DISBURSEMENT
 ]
 
+reclamiation_filters = [
```
Comment (tristanhcole) [inline app/server/utils/metrics/filters.py:146]: nit: spelling should be "reclamation"

File: app/server/utils/metrics/transfer_stats.py [hunk 2]
```diff
@@ -19,30 +19,48 @@ def __init__(self, group_strategy, timeseries_unit = 'day', token=None):
...
         self.metrics.append(metric.Metric(
             metric_name='total_distributed',
-            query=total_distributed_query,
+            query=total_amount_query,
             object_model=CreditTransfer,
             stock_filters=[filters.disbursement_filters],
             caching_combinatory_strategy=metrics_cache.SUM,
             filterable_by=self.filterable_attributes,
             bypass_user_filters=True,
         ))
```
Comment (tristanhcole) [inline app/server/utils/metrics/transfer_stats.py:31]: If it's the total amount that should be returned, shouldn't the stock filter `filters.disbursement_filters` be removed? Sorry haven't been involved in backend metrics so trying to wrap my head around queries

**Review by tristanhcole — APPROVED** [review]
Tested,LGTM! One nit, one question to be resolved 😄

File: app/server/utils/metrics/transfer_stats.py (hunk 2 above)
Comment (enjeyw) [reply app/server/utils/metrics/transfer_stats.py:31]: oh `total_amount_query` is the base query that all metrics of this type use. So u can see it's reused agian below. (If u look at the lines removed, they were all just copy pastes, so this cleans up)

```
        total_distributed_query = db.session.query(func.sum(CreditTransfer.transfer_amount).label('total'))
        total_spent_query       = db.session.query(func.sum(CreditTransfer.transfer_amount).label('total'))
        total_exchanged_query   = db.session.query(func.sum(CreditTransfer.transfer_amount).label('total'))
```

File: app/server/utils/metrics/filters.py (hunk 1 above)
Comment (enjeyw) [reply app/server/utils/metrics/filters.py:146]: i spal gud.

**Pull request merged**